src/models/phoneNumber.model.ts

```diff
@@ -0,0 +1,19 @@
+import { model, Schema, Model } from 'mongoose';
+
+export interface IPhoneNumber extends Document {
+  country_id: string;
+  phone_number: string;
+  is_active: boolean;
+}
+
+const PhoneNumberSchema: Schema = new Schema(
+  {
+    country_id: { type: Schema.Types.ObjectId, required: true, ref: 'Country' },
+    phone_number: { type: String, required: true, trim: true },
+    is_active: { type: Boolean, default: false },
+  },
+  { timestamps: true }
+);
+
+const PhoneNumber: Model<IPhoneNumber> = model('PhoneNumber', PhoneNumberSchema);
```

Comment:
Dùng PhoneNumber -> mongosee sẽ support lưu với tên table là phonenumbers. Nhìn hơi tù.
Có thể dùng thêm tham số thứ 3 để replace table nếu muốn nhé người ae
 ```suggestion
const PhoneNumber: Model<IPhoneNumber> = model('PhoneNumber', PhoneNumberSchema, 'phone_numbers');
```